readme.org

```diff
@@ -189,9 +189,12 @@ more.
        - Tab key
          - Tab key is recognized as ~<tab>~ in GUI and ~TAB~ in terminal.
            ~TAB~ equals ~C-i~.
-         - Since ~C-i~ is bound to jumping forward for vim compatibility,
-           bind ~g[~ and ~g]~ to the functions that Shift+Tab and Tab
-           are bound to on GUI for terminal compatibility.
+         - ~C-i~ is bound to jumping forward for vim compatibility.
+           If Shift+Tab is not relevant, just bind ~g TAB~ to the function
+           that Tab is bound to. If Shift+Tab is relevant,
+           bind ~g]~ and ~g TAB~ to the function that Tab is bound to,
+           and bind ~g[~ or ~g}~ to the function that Shift+Tab is bound to
```

Comment:
Why `g}`?

Reply:
Because Shift+Tab may not be functionally related to Tab in some modes such as org mode.